### ISSUE TITLE
The user is redirected to home page 

### ISSUE THREAD

**Issue by ovidiuboca1** [mozilla/sumo-project#308]
Prerequisites: you need to be logged out

STR:

1. Go to https://support.allizom.org/en-US/get-involved/questions 
              https://support.allizom.org/en-US/get-involved/kb
              https://support.allizom.org/en-US/get-involved/l10n
2. Click on the "Sign me up" button 
3. Sign in 

ER: After the login, you are redirected to the same page from where you started the login process. 
AR: You are redirected to the home page. Screen recording - https://streamable.com/8vb34

**Comment by akatsoulas** [comment]
PR https://github.com/mozilla/kitsune/pull/4269

**Comment by gauravghati** [comment]
Hey @akatsoulas, I would like to work on this issue, will it be okay if I redirect the user to [https://support.allizom.org/en-US/](https://support.allizom.org/en-US/) after signin?

**Comment by LeoMcA** [comment]
Hey @gauravghati, https://support.allizom.org/en-US/ is where we don't want them to end up, instead we want users to end up on the page they started on.

So, for example, if they click the button on https://support.allizom.org/en-US/get-involved/kb, they end up back on https://support.allizom.org/en-US/get-involved/kb after signing in.

@akatsoulas I noticed you closed a PR for this without merging: https://github.com/mozilla/kitsune/pull/4269. Was there a problem with that approach which @gauravghati might be able to help us solve?

**Comment by gauravghati** [comment]
> So, for example, if they click the button on https://support.allizom.org/en-US/get-involved/kb, they end up back on https://support.allizom.org/en-US/get-involved/kb after signing in.
> 
> @akatsoulas I noticed you closed a PR for this without merging: [mozilla/kitsune#4269](https://github.com/mozilla/kitsune/pull/4269). Was there a problem with that approach which @gauravghati might be able to help us solve?

Okay, I understand the bug. I'll try to solve it.

**Comment by akatsoulas** [comment]
I don't recall the specifics but the approach should be valid and the behavior is the one you described. Thank you @gauravghati for your interest into this issue!

**Comment by mirunacurtean** [comment]
Verified across Windows 10 pro 64-bit, mscOS Mojave 10.14.6 and Android 9.0.0 on FF Release, Chrome and Safari on en-us and FR locales. 
Could not verify this works when signing up due to #563 and I've made a note to re-test this scenario when that is fixed, but sign in was successful and redirect is performed to the same page. 
Initially I could still see the 'Sign me up' button after login. but now I can no longer reproduce this on older or new profiles. 
![Sign me up button still available after login](https://user-images.githubusercontent.com/53306802/97726915-6bdf8180-1ad8-11eb-936d-3206b21bf6c6.gif)
I'll keep an eye on it and if it becomes reproducible again I will log a new issue.
